core/controllers/reader.py

```diff
@@ -570,11 +570,11 @@ def post(self, exploration_id):
         if user_id and collection_id:
             collection_services.record_played_exploration_in_collection_context(
                 user_id, collection_id, exploration_id)
-            collections_left_to_complete = (
-                collection_services.get_next_exploration_ids_to_complete_by_user( # pylint: disable=line-too-long
+            exploration_left_to_complete = (
```

Comment:
Isn't this the wrong name? next_exp_id_to_complete would be better?

Reply:
Done.